packages/core/src/actions/getTransactionConfirmations.ts

```diff
@@ -0,0 +1,36 @@
+import {
+  type GetTransactionConfirmationsErrorType as viem_GetTransactionConfirmationsErrorType,
+  type GetTransactionConfirmationsParameters as viem_GetTransactionConfirmationsParameters,
+  type GetTransactionConfirmationsReturnType as viem_GetTransactionConfirmationsReturnType,
+  getTransactionConfirmations as viem_getTransactionConfirmations,
+} from 'viem/actions'
+
+import { type Config } from '../createConfig.js'
+import { type ChainIdParameter } from '../types/properties.js'
+import { type Evaluate } from '../types/utils.js'
+
+export type GetTransactionConfirmationsParameters<
+  config extends Config = Config,
+  chainId extends config['chains'][number]['id'] = config['chains'][number]['id'],
+> = Evaluate<
+  viem_GetTransactionConfirmationsParameters & ChainIdParameter<config, chainId>
```

Comment:
Need to wire up generics for [`viem_GetTransactionConfirmationsParameters`](https://github.com/wevm/viem/blob/b730ef3d21a63f443e1a1078e90f27af6c662cec/src/actions/public/getTransactionConfirmations.ts#L19)